web/tests/test_grade_link_feature.py

```
@@ -0,0 +1,672 @@
+import pytest
```

Ruff (0.8.2)

`pytest` imported but unused

Remove unused import: `pytest`

(F401)